usecases/objects/auto_schema.go

```diff
@@ -73,10 +89,11 @@ func (m *autoSchemaManager) autoSchema(ctx context.Context, principal *models.Pr
 
 		object.Class = schema.UppercaseClassName(object.Class)
 
-		schemaClass, schemaVersion, err := m.schemaManager.GetCachedClass(ctx, principal, object.Class)
-		if err != nil {
-			return 0, err
-		}
+		vclass := vclasses[schema.UppercaseClassName(object.Class)]
```

Comment:
if class is taken from `vclasses` map now, it has to be updated whenever it changes (therefore `classcache.RemoveClassFromContext` calls further in the code, so `m.schemaManager.GetCachedClass` would fetch updated class).

I see `vclasses` updated when new class is created, but no when new properties are added to the class.

Reply:
good catch!